LoginController.java

```diff
@@ -0,0 +1,62 @@
+package com.application.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import com.application.model.Teacher;
+import com.application.model.User;
+import com.application.services.TeacherService;
+import com.application.services.UserService;
+
+@RestController
+public class LoginController 
+{
+	@Autowired
+	private UserService userService;
+	
+	@Autowired
+	private TeacherService teacherService;
+	
+	//welcome page

```

Comment:
@GetMapping("/")
    public String welcomeMessage()
    {
    	return "Welcome to Virtual Classroom !!!";
    }

Reply:
I edited my Login Controller source file for Mapping the welcome page.
Could you recheck review my code